src/main/kotlin/com/esops/service/UserService.kt

```diff
@@ -265,7 +264,7 @@ class UserService {
 //        val timeDiffInDays: Int = ((currentTime - i.addedAt) / nanoSecsIn1Day).toInt()
             val timeDiffInSecs: Int = ((currentTime - i.addedAt) / milliSecsIn1Sec).toInt()
             for (day in 0 until i.dividedInventory.size) {
-                if ((day + 1) * duration < timeDiffInSecs) {
+                if ((day + 1) * duration <= timeDiffInSecs) {
```

Comment:
This cannot be an if condition. It is unreadable. What does (day + 1) * Duration mean. What does this if do?